code/game/machinery/camera/camera.dm

```diff
@@ -245,24 +246,23 @@
 	set_status(!src.status)
 	if (!(src.status))
 		if(user)
-			visible_message(SPAN_NOTICE(" [user] has deactivated [src]!"))
+			balloon_alert_to_viewers("[user] has deactivated [src]!")
```

Comment:
wrong usage of balloon alerts

Reply:
whats the correct use

Reply:
Not for messages X did Y

Reply:
[Read TGstation's guide on how to use balloon alerts.](https://hackmd.io/@tgstation/SJIle5wYd#balloon_alertballoon_alert_to_viewers)

Reply:
```suggestion
			balloon_alert_to_viewers("camera deactivated")
```